api/part2/openapi/schemas/json/dataStream.json

```diff
@@ -72,17 +95,39 @@
         },
         "type": {
           "type": "string",
-          "enum": ["status", "observation"]
+          "enum": [
+            "status",
+            "observation"
+          ]
         },
         "resultType": {
-          "type": "string",
-          "enum": ["measure", "vector", "record", "coverage", "complex"],
-          "readOnly": true
```

Comment:
I think we should still leave the auto-generated properties marked as read-only, even though we also allow 'null' when no obs are associated yet.

Reply:
Sorry I didn´t see the `readOnly` flag is still present inside the `oneOf`.
But isn't it better to keep it at the root so it applies even for the null case?

Reply:
>  But isn't it better to keep it at the root so it applies even for the null case?

yes.

i will fix it - apparently i was a not thorough enough while  copy-pasting and the `readonly` is ended up all over the place.